collect_app/src/androidTest/java/org/odk/collect/android/support/CopyFormRule.java

```diff
@@ -71,6 +73,7 @@ public void evaluate() throws Throwable {
             } catch (IllegalStateException e) {
                 component.projectImporter().importDemoProject();
                 component.currentProjectProvider().setCurrentProject(ProjectImporter.DEMO_PROJECT_ID);
+                projectCreated = true;
```

Comment:
Ideally we'd just get rid of `CopyFormRule` but that would be a lot of energy right now.